### PR TITLE
Restructure of file

### LINKED ISSUE #16: Restructure files
Footer.tsx now routes to App.tsx, which I suppose is not good since App.tsx is suppose to be the parent component.

Suggestion: Rename the current App.tsx to something else, for example, ChattApp.tsx, create a new App.tsx that renders Tabbar.tsx. However I don't know what happens with state and things like this. Discuss with Magnus before doing this.

### PR DESCRIPTION
**Pull request by Monjola**
App -> ChatApp
New App renders Tabbar that routes to ChatApp and the other tabs

### CODE REVIEW

File: src/components/ChatApp/ChatApp.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  }
+
+  const subject = subjects.find((s) => s.id === channelId);
+
+  const showPopup = !name && !loading;
+  const showSubjectList = !subject && !showPopup && !loading;
+  const blurChat = showPopup || showSubjectList || loading;
+
+  return (
+    //<ErrorBoundary>
```
Comment (iRobsc) [inline src/components/ChatApp/ChatApp.tsx]: Same with this line.

File: src/components/ChatApp/ChatApp.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react';
+import Popup from '../Popup';
+import SubjectList from '../SubjectList';
+import * as api from '../../api';
+import Chat from '../Chat';
+import LoadingIndicator from '../LoadingIndicator';
+import { useCookie } from '../../useCookie';
+//import ErrorBoundary from './ErrorBoundary';
```
Comment (iRobsc) [inline src/components/ChatApp/ChatApp.tsx]: It's not really important, but having commented code can make files cluttered fast. So I usually avoid having them commited to the repositories but you can decide what you want to do with this.

**Review by iRobsc — APPROVED** [review]
I approve this pull request and it looks really good but left some small comments about commenting old code. This is mostly subjective so you can do whatever you think is correct.

File: src/components/ChatApp/ChatApp.tsx (hunk 2 above)
Comment (Monjola) [reply src/components/ChatApp/ChatApp.tsx]: removed

File: src/components/ChatApp/ChatApp.tsx (hunk 1 above)
Comment (Monjola) [reply src/components/ChatApp/ChatApp.tsx]: removed

**Pull request merged**